### ISSUE TITLE
MVT Layer spec v2?

### ISSUE THREAD

**Issue by chriswhong** [CartoDB/Windshaft#525]
Hi, I am seeing this error when rendering carto vector tiles with MapboxGL v0.31.0:

![facilities_explorer_and_slack](https://cloud.githubusercontent.com/assets/1833820/22342627/0bbb3588-e3c3-11e6-963c-248b8632ccfc.png)

Just wondering if there is any plan to update the vector tiles.  They don't seem to have any rendering errors, just this warning popping up in the JS console.  Thanks!!

**Comment by arethasamuel** [comment]
1) Sorry, this is unrelated to your error, but were you able to display line features? through mvt?  i get some funny looking lines, polygons and point features are showing correctly.

2) Also, were you also able to display attributes as well? (send attribute information along with the vector data)

(i also got this warning myself)

**Comment by chriswhong** [comment]
We have only been using points and polygons. I actually did try to set up a line feature and got a lot of weird noise on the screen (lots of horizontal lines styled the way I wanted my line feature styled)

**Comment by Algunenano** [comment]
This should have been addressed with carto/node-mapnik 3.6.2-carto.8 (Windshaft 4.7.1):

```json
[
  {
    "version": 2, <<<<<<<<<<<<<<<<<<<<<<<<<<<
    "name": "field_shapes",
    "extent": 4096,
    "features": [
      {
        "type": 1,
        "id": 1,
        "properties": {
          "cartodb_id": 1,
          "data": 0,
          "status2": false
        },
        "geometry": [
          9,
          682,
          8178
        ]
      },
      {
        "type": 1,
        "id": 2,
        "properties": {
          "cartodb_id": 2,
          "data": 0,
          "status2": false
        },
        "geometry": [
          9,
          4096,
          4096
        ]
      }
    ]
  }
]
```
